docs/permissions/limitations.md

```diff
@@ -217,6 +217,24 @@ Beyond that the following Limitations are available:
 
 - [Product Type](limitation_reference.md#product-type-limitation)
 
+#### Cart
+
+`cart/create`:
+
+- [CartOwner](limitation_reference.md#cartowner-limitation)
+
+`cart/view`:
+
+- [CartOwner](limitation_reference.md#cartowner-limitation)
+
+`cart/edit`:
+
+- [CartOwner](limitation_reference.md#cartowner-limitation)
+
+`cart/delete`:
+
+- [CartOwner](limitation_reference.md#cartowner-limitation)
```

Comment:
This doesn't seem right. Shouldn't you reference dedicated pages instead of `CartOwner` everywhere?

Reply:
that's how it is done for other permissions, when the limitation repeats

Reply:
Ok, thanks for clarification.